js/media/fieldmanager-media.js

```diff
@@ -57,7 +57,23 @@ $( document ).on( 'click', '.fm-media-button', function( event ) {
 		$wrapper.html( preview ).trigger( 'fieldmanager_media_preview', [ $wrapper, attachment, wp ] );
 	});
 
+	// Select the attachment when the frame opens
+	fm_media_frame[ $el.attr('id') ].on( 'open', function() {
+		// Select the current attachment inside the frame
+		var selection = fm_media_frame[ $el.attr('id') ].state().get('selection'),
+			id = $el.parent().find('.fm-media-id').val(),
```

Comment:
just want to confirm, there's no way there would be more than one `.fm-media-id` in `$el`'s parent, right?